requirements-common.txt

```diff
@@ -8,12 +8,10 @@ py-cpuinfo
 transformers >= 4.48.2  # Required for Bamba model and Transformers backend.
 tokenizers >= 0.19.1  # Required for Llama 3.
 protobuf # Required by LlamaTokenizer.
-fastapi >= 0.107.0, < 0.113.0; python_version < '3.9'
-fastapi >= 0.107.0, != 0.113.*, != 0.114.0; python_version >= '3.9'
+fastapi[standard] >= 0.107.0, < 0.113.0; python_version < '3.9'
```

Comment:
By including the `standard` extras, we include `python-multipart` package which is required for [`UploadFile`](https://fastapi.tiangolo.com/reference/uploadfile/) and we include `uvicorn` as well (see deletion below).